collection-scripts/gather

```diff
@@ -93,6 +99,7 @@ Options:
   -cl, --ceph-logs          Collect ceph daemon, kernel, journal logs and crash reports
   -ns, --namespaced         Collect namespaced resources
   -cs, --clusterscoped      Collect clusterscoped resources
+  -m,  --minimal            Collect only relevant CRDs and CSVs
```

Comment:
A more specific description would be nice. "only relevant" sounds a bit vague?

Reply:
Done